src/server/rest/v1/service/UserService.ts

```diff
@@ -672,44 +672,46 @@ export default class UserService {
           passwordBlockedUntil: null
         });
     }
-    // Only Admin can save these data
-    if (Authorizations.isAdmin(req.user) || Authorizations.isSuperAdmin(req.user)) {
-      // Save User Status
-      if (newUser.status) {
-        await UserStorage.saveUserStatus(req.tenant, newUser.id, newUser.status);
-      }
-      // Save User Role
-      if (newUser.role) {
-        await UserStorage.saveUserRole(req.tenant, newUser.id, newUser.role);
+    // Save User Status
+    if (newUser.status) {
```

Comment:
Auth must be checked I think to avoid creating an Admin user via Postman, right?

Reply:
I think the new BasicUser assert will prevent that.